kihuha/iReporter
# #163398303 - Admin update status

#### What does this PR do?
Contains functionality that allows an admin to change the status of a report.

#### Description of the task to be completed
- Add new dropdown DOM element if the user logged in is an admin
- Make a patch request with the input values afforded

#### How should this be manually tested
An admin should be created by an already existing admin. Once logged in with the `isadmin=true`, click on the update button of a record. The page redirects to the update-report page which contains a form pre-filled with the existent details. The admin can click on the dropdown labelled type and change the status of a record

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/163398303
